AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2180,14 +2202,15 @@ - (void)setInterfaceState:(ASInterfaceState)newState
   BOOL wasVisible = ASInterfaceStateIncludesVisible(oldState);
 
   if (nowVisible != wasVisible) {
-    [self visibilityDidChange:nowVisible];
+    [self visibileStateDidChange:nowVisible];
```

Comment:
Here we can call both methods back to back, unconditionally, as long as the base class has a blank implementation of both.